src/readJsonLines.js

```diff
@@ -1,6 +1,8 @@
 function readJsonLines(string) {
   return string.split('\n')
     .filter(line => line)
+    .filter(line => line.startsWith('{')
```

Comment:
a more reliable way for checking valid JSON is to try to parse it and handle an error in a `catch` block

Reply:
okay, will check this